starter_kits/TypeScript/hlt/Game.ts

```diff
@@ -0,0 +1,140 @@
+import { createInterface } from "readline";
+import { Constants } from "./Constants";
+import { Logging } from "./Logging";
+import { GameMap } from "./GameMap";
+import { Player } from "./Player";
+
+
+export class Game {
+    turnNumber: number;
+
+    _getLine: () => Promise<string>;
+    public myId: number = 0;
+    public players = new Map<number, Player>();
+    public me: Player | undefined = undefined;
```

Comment:
You can do
`me ?: Player`

You don’t have to explicitly have `undefined` with the `?` operator.

Reply:
Good point. I'll update all.